auto3dseg/algorithm_templates/swinunetr/configs/transforms_infer.yaml

```diff
@@ -3,21 +3,22 @@ transforms_infer:
   _target_: Compose
   transforms:
   - _target_: LoadImaged
-    keys: '@image_key'
+    keys: "@image_key"
+    image_only: True
   - _target_: EnsureChannelFirstd
-    keys: '@image_key'
+    keys: "@image_key"
+  - PLACEHOLDER_INTENSITY_NORMALIZATION
   - _target_: Orientationd
+    keys: "@image_key"
     axcodes: RAS
-    keys: '@image_key'
   - _target_: Spacingd
-    align_corners: true
-    keys: '@image_key'
+    keys: "@image_key"
+    pixdim: "$@transforms#resample_to_spacing"
```

Comment:
Hi @heyufan1995 , can you confirm `resample_to_spacing` is under `transforms`
I found it under the root level in `algo.py`
```
hyper_parameters.update({"resample_to_spacing": spacing})
```

Reply:
yes it's defined twice in config
transforms:
  resample_to_spacing: "$@resample_to_spacing"
  lazy_resampling: true
resample_to_spacing: null